src/model/predictor.py

```
@@ -110,21 +114,24 @@ def save_prediction_to_excel(
 
-        df = pd.DataFrame({
-            'time': time_points,
-            'value': value_points
-        })
+        df = pd.DataFrame({"time": time_points, "value": value_points})
 
-        output_path = os.path.join(config.OUTPUT_DIR, filename)
-        df.to_excel(output_path, index=False, engine='openpyxl')
+        # 拡張子の保証
+        safe_name = filename if filename.lower().endswith(".xlsx") else f"{filename}.xlsx"
+        # ディレクトリ作成（存在しない場合）
```

Ruff (0.12.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)